src/adapter/service-worker/handler.test.ts

```diff
@@ -2,6 +2,30 @@ import { Hono } from '../../hono'
 import { handle } from './handler'
 import type { FetchEvent } from './types'
 
+beforeAll(() => {
+  // fetch errors when it's not bound to globalThis in service worker
+  // set a fetch stub to emulate that behavior
+  vi.stubGlobal(
+    'fetch',
+    function fetch(this: undefined | typeof globalThis, arg0: string | Request) {
+      if (this !== globalThis) {
+        const error = new Error(
+          'Failed to execute \'fetch\' on \'WorkerGlobalScope\': Illegal invocation'
```

Comment:
The formatter and linter is conflicting in this line and that's the reason why the CI is failing.
Prettier tries to change this into `"Failed to execute 'fetch' on 'WorkerGlobalScope': Illegal invocation"` and ESLint's quotes turns this into `'Failed to execute \'fetch\' on \'WorkerGlobalScope\': Illegal invocation'`.

Reply:
Can you add the following above that line? It's okay to avoid the error with this.

```ts
// eslint-disable-next-line quotes
```

Reply:
I added the ignore comment 👍